integration-tests/maven/src/it/setup-on-existing-pom-it/invoker.properties

```diff
@@ -1 +1 @@
-invoker.goals=io.quarkus:quarkus-maven-plugin:${project.version}:create
+invoker.goals=io.quarkus:quarkus-maven-plugin:${project.version}:create -DplatformArtifactId=quarkus-bom -DplatformVersion=${project.version}
```

Comment:
these explicit set of platform properties is to just make current tests pass using the "built-in" platform catalog, right ? 

could we add some tests (possibly just unit tests) for the various combinations of the resolution logic since there seem to be quite a lot of different execution paths in the logic ?

Reply:
> these explicit set of platform properties is to just make current tests pass using the "built-in" platform catalog, right ?

Right.

> could we add some tests (possibly just unit tests) for the various combinations of the resolution logic since there seem to be quite a lot of different execution paths in the logic ?

I added tests for the resolver in the tools resolver.